.github/pull_request_template.md

```diff
@@ -1,33 +1,32 @@
 <!--
-Thank you for contributing to **OceanBase**! 
 
-**If this pull request have a significant impact, please make sure you have discussed with OceanBase group.**
--->
+Thank you for contributing to OceanBase! Please feel free to ping the maintainers for the review!
 
-### Task Description
+We recommend to write commit message and pull request title in the format of conventional commits:
 
-<!--
-The problem you resolved by this pull request.
-You can link the issue via the "close #xxx" or "ref #xxx".
--->
+```
+<type>[optional scope]: <description>
+```
 
-### Solution Description
+For more details about conventional commits, see https://www.conventionalcommits.org.
 
-<!-- Please clearly and consice descipt the solution. -->
+-->
 
-### Passed Regressions
+## Description
 
-<!-- Unittest, mysql test or test it manually? -->
+<!-- Describe the purpose of this pull request. -->
 
-### Upgrade Compatibility
+This pull request ...
 
-<!-- Please make sure this is compatible with old version or you should give us upgrading solution. -->
+<!-- If this pull request resolves an issue, reference it with the issue number. -->
 
-### Other Information
+Close #
 
-<!-- Any information helping to review this pull request. -->
+## Checklist
 
-### Release Note
-<!--
-A concise release note can help users to understand how your pull request makes difference.
--->
+<!-- Change `[ ]` to `[x]` to mark the item is checked.-->
```

Comment:
The checklist is too long and the spec is not concise, so I think the users will read the message impatiently.

Reply:
Removed, now at it‘s more simpler.